charts/istio-gateway/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for common.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+gatewayClass: "istio-ingressgateway"
+replicaCount: 2
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using Release.Name
+  name: ""
+
+resources:
+  limits:
+    cpu: 2000m
+    memory: 1024Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
Also decided a while ago to lockstep memory requests to memory limits. Do we need to discuss on the why?